module/functions/Connect-DataverseEnvironment.ps1

```diff
@@ -46,25 +75,38 @@ function Connect-DataverseEnvironment
 
     $safeEnvironmentUrl = $EnvironmentUrl.TrimEnd('/')
 
-    $options = [Microsoft.Identity.Client.PublicClientApplicationOptions]@{
-        ClientId = $ClientId
-        TenantId = $TenantId
-        # AadAuthorityAudience = [Microsoft.Identity.Client.AadAuthorityAudience]::AzureAdMultipleOrgs
-        RedirectUri = "http://localhost"
+    # Acquire the token for CI/CD scenarios
+    if ($env:AZURE_CLIENT_ID -and $env:AZURE_CLIENT_SECRET -and $env:AZURE_TENANT_ID)
+    {
+        Write-Host "Attemping authentication via environment variables [ClientId=$env:AZURE_CLIENT_ID]"
+        $authResult = Get-MsalToken -ClientId $env:AZURE_CLIENT_ID `
+                                    -Scopes $authScope `
```

Comment:
Where does `$authScope` come from?

Reply:
Looks like the corresponding code has been deleted?

Reply:
Good catch, it got renamed.